.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,3 +1,9 @@
+<!--
```

Comment:
This might need to be last in the file, not sure if the github parser for templates allows this.